dd-sdk-android-ndk/src/main/cpp/datadog-native-lib.cpp

```diff
@@ -17,15 +17,30 @@
 #include "utils/datetime.h"
 #include "utils/fileutils.h"
 #include "utils/signal-monitor.h"
+#include "utils/stringutils.h"
+
+static struct rumcontext {
+    char application_id[200];
```

Comment:
Where does the `200` come from? Is it worth putting in a constant somewhere, e.g.
```
static const size_t MAX_ID_SIZE = 200;
```